json-endpoints/jetpack/class.jetpack-json-api-plugins-install-endpoint.php

```diff
@@ -38,6 +38,12 @@ protected function install() {
 		}
 
 		if ( ! $this->bulk && isset( $error ) ) {
+
+			if ( $error_code === 'download_failed' && $error === 'Download failed.' ) {
```

Comment:
True. good catch. I was also checking `$error` because I'm not sure if the error code `download_failed` can be returned for multiple different reasons. I'll investigate and fix.